nextflow_schema.json

```diff
@@ -39,106 +43,88 @@
+                    "fa_icon": "fas fa-search-plus"
                 },
-                "slamseq": {
+                "pbat": {
                     "type": "boolean",
-                    "description": "Run bismark in SLAM-seq mode.",
-                    "fa_icon": "fas fa-wave-square",
-                    "help_text": "Specify to run Bismark with the `--slam` flag to run bismark in [SLAM-seq mode](https://github.com/FelixKrueger/Bismark/blob/master/CHANGELOG.md#slam-seq-mode) \n\n> NB: Only works with when using the `bismark_hisat` aligner (`--aligner bismark_hisat`)"
-                },
-                "em_seq": {
-                    "type": "string",
-                    "fa_icon": "fas fa-cubes",
-                    "description": "Preset for EM-seq libraries.",
-                    "help_text": "Equivalent to `--clip_r1 8` `--clip_r2 8` `--three_prime_clip_r1 8` `--three_prime_clip_r2 8`.\n\nAlso sets the `--maxins` flag to `1000` for Bismark."
+                    "help_text": "Specifying `--rrbs` will pass on the `--rrbs` parameter to TrimGalore! See the [TrimGalore! documentation](https://github.com/FelixKrueger/TrimGalore/blob/master/Docs/Trim_Galore_User_Guide.md#rrbs-specific-options-mspi-digested-material) to read more about the effects of this option.\n\nThis parameter also makes the pipeline skip the deduplication step.\n",
+                    "fa_icon": "fas fa-search-plus"
                 },
                 "single_cell": {
                     "type": "boolean",
-                    "fa_icon": "fas fa-cut",
-                    "description": "Trimming preset for single-cell bisulfite libraries.",
-                    "help_text": "Equivalent to `--clip_r1 6` `--clip_r2 6` `--three_prime_clip_r1 6` `--three_prime_clip_r2 6`.\n\nAlso sets the `--non_directional` flag for Bismark."
+                    "help_text": "Specifying `--single_cell` will pass on the `--single_cell` parameter to TrimGalore!\n\nNote that the `--single_cell` and `--zymo` parameters both set the `--non_directional` workflow flag automatically.\n",
+                    "fa_icon": "fas fa-search-plus"
+                },
+                "epignome": {
+                    "type": "boolean",
+                    "help_text": "Specifying `--epigenome` will pass on the `--epigenome` parameter to TrimGalore!\n",
+                    "fa_icon": "fas fa-search-plus"
                 },
                 "accel": {
                     "type": "boolean",
-                    "fa_icon": "fas fa-cut",
-                    "help_text": "Equivalent to `--clip_r1 10` `--clip_r2 15` `--three_prime_clip_r1 10` `--three_prime_clip_r2 10`",
-                    "description": "Trimming preset for the Accel kit."
+                    "help_text": "Specifying `--accel` will pass on the `--accel` parameter to TrimGalore! ",
```

Comment:
This is not the case - the original help text is correct here. TrimGalore doesn't have such a parameter. This is used in the following code:

https://github.com/nf-core/methylseq/blob/15cc666feb0b4085c30108a10e7ab120b9907ac8/main.nf#L125-L130